orc8r/cloud/helm/orc8r/values.yaml

```diff
@@ -52,6 +52,34 @@ secret:
     orc8r: orc8r-secrets-configs-orc8r
   envdir: orc8r-secrets-envdir
 
+# certs sub-chart configuration.
+certs:
+  create: false
+
+  dnsDomain: localhost
+
+  # Following durations is set on these certificates:
+  # - nms-certificate
+  # - controller-certificate
+  # - admin-operator-certificate
+  # 10 years = 87600h
+  duration: 87600h
```

Comment:
It's 10 years

https://github.com/magma/magma/blob/master/orc8r/cloud/deploy/scripts/self_sign_certs.sh#L42

Reply:
Oh wow I 100% thought it was 1 year. Good to know

Reply:
I'm fine with leaving default relatively high, but since auto-renew should work well, something like 1 or 3 years for these non-root certs also seems reasonable. Those original values were chosen under the expectation that it would be a hassle to manually renew

That aside, we do want to rotate the bootstrapper.key frequently, e.g. on a monthly basis, since it's being used relatively frequently (once every 4 days x number of gateways pointing to the Orc8r)

Reply:
Agree with @hcgatewood on the need for rotation. The CA/Browser forum officially [adopted ](https://thehackernews.com/2020/09/ssl-tls-certificate-validity-398.html)398 days as the time limit for expiring security certificates. We should aspire to meet this goal with the option to automatically check and renew security certificates, which cannot happen with self-signed certificates since they do not share ancestry with any other certificates

Reply:
I have added a new `bootstrapper-certificate`, which will create a new RSA key for us and also added `rotationPolicy: Always` to it.